core/server/master/src/main/java/alluxio/master/block/BlockMasterWorkerServiceHandler.java

```diff
@@ -134,6 +136,13 @@ public RegisterWorkerTResponse registerWorker(final long workerId,
       public RegisterWorkerTResponse call() throws AlluxioException {
         mBlockMaster.workerRegister(workerId, storageTiers, totalBytesOnTiers, usedBytesOnTiers,
             currentBlocksOnTiers);
+        if (options.isSetConfigList()) {
```

Comment:
I think this processing should be in `BlockMaster`, instead of this rpc handler class.